src/aics-image-viewer/components/App/index.tsx

```diff
@@ -325,9 +325,19 @@ const App: React.FC<AppProps> = (props) => {
     const grouping = makeChannelIndexGrouping(channelNames, getCurrentViewerChannelSettings());
     setChannelGroupedByType(grouping);
 
-    const settingsAreEqual = channelNames.every((name, idx) => name === channelSettings[idx]?.name);
-    if (settingsAreEqual) {
-      return channelSettings;
+    // compare each channel's new displayName to the old displayNames currently in state:
+    // same number of channels, and each channel has same displayName
+    const allNamesAreEqual = channelNames.every((name, idx) => {
+      const displayName = getDisplayName(name, idx, getCurrentViewerChannelSettings());
+      return displayName === channelSettings[idx]?.displayName;
+    });
+
+    if (allNamesAreEqual) {
+      const newChannelSettings = channelNames.map((channel, index) => {
+        return { ...channelSettings[index], name: channel };
+      });
+      setChannelSettings(newChannelSettings);
+      return newChannelSettings;
```

Comment:
The above lines use a new criterion to test whether to use old or new/defaulted channel settings.  It uses the remapped display name after the getCurrentViewerChannelSettings are applied, instead of the old way of just using the raw native channel names..

Reply:
This might not apply,  but does this mean that the current ramp/control points will be preserved too? Could that cause a volume to look super blown out when switching?